### PR TITLE
[#93] CNF validation and service flag fixed (NOT merge)

### PR DESCRIPTION
**Pull request by MiguelRivasQuobis**
Now, the validator works although the descriptor contains cloudnative_deployment_units.

### CODE REVIEW

**Comment by mpeuster** [comment]
Is there a reason to `force-push`? Doesn't this make this entire history of your fork incompatible to the `master`?

**Comment by anapolg** [comment]
> Is there a reason to `force-push`? Doesn't this make this entire history of your fork incompatible to the `master`?

Hi Manuel, the reason is that he wanted to do some checks on his code using the Jenkins, which is only pointing this repository and not the forks, but it was not stable nor final code for a commit. Anyhow, we will reduce the use of force push from now on. Sorry for the amount of notifications you have received.

**Comment by mpeuster** [comment]
Ah ok. But Jenkins should always run if you open a Pull request. No matter from which branch this PR is opened. Eg. you can open a PR from `user-fork:feature-branch` to `tng-sdk-validate:master`... this should trigger jenkins as usual.  if you only want to test you can write in the PR "do not merge" so people know that hey shouldn't hit "merge". :-)

File: src/tngsdk/validation/cli.py [hunk 1]
```diff
@@ -88,12 +88,19 @@ def dispatch(args, validator):
...
-            if len(validator.customErrors) == 0:
-                print("No errors found in the Service validation")
-            else:
-                print("Errors in custom rules validation")
+        #Using the following statements it is possible distinguish between using --dpath or not
+        #TODO check if the user MUST put '/' at the end of path in the case of using ---dpath
+        if args.dpath is None:
+            nsd_path = args.nsd
+        else:
+            nsd_path = args.dpath+args.nsd
```
Comment (mpeuster) [inline src/tngsdk/validation/cli.py]: `os.path.join(x, y)` would make this more stable, also solve `/` issues

**Comment by anapolg** [comment]
> you can open a PR from user-fork:feature-branch to tng-sdk-validate:master... this should trigger jenkins as usual

Yes, that's what he is doing. He is doing force push over his fork and, the PR opened gets updated with every change. The Jenkins is launched every time like this and he is able to test.

>  if you only want to test you can write in the PR "do not merge" so people know that hey shouldn't hit "merge".

Completely right. We'll do that :)

File: src/tngsdk/validation/cli.py (hunk 1 above)
Comment (anapolg) [reply src/tngsdk/validation/cli.py]: @MiguelRivasQuobis please take this into account :)

**Comment by MiguelRivasQuobis** [comment]
Okey, i will include it in the next update

**Comment by mpeuster** [comment]
But why do you need to force push for this? A normal push should have the same effect?

**Pull request closed without merging**